packages/create-app/src/index.ts

```diff
@@ -210,7 +210,8 @@ async function main() {
         console.log(chalk.cyan(`  cd ${directory}`));
         console.log(chalk.cyan('  npm install'));
         if (editorResponse.includeEditor) {
-          console.log(chalk.cyan('  cd editor && npm install'));
+          console.log(chalk.cyan(` cd ${path.join(__dirname, '..', 'components', 'editor')}`));          
```

Comment:
why do you need this logic? cant we just show:

`cd ${directory}/editor`

Reply:
I think the editor is in src/components/editor location in this case.